hw/ip/kmac/data/kmac.hjson

```diff
@@ -217,6 +217,58 @@
       package: "lc_ctrl_pkg"
     }
   ]
+  features: [
```

Comment:
It seems this is a new hjson field. How is it used? Can it be documented somewhere, like in https://opentitan.org/book/doc/contributing/hw/comportability/index.html.

Reply:
Hi @matutem, I started documenting the proposal for this hjson field in #19444.  I can update the comportability once we agree on the format and use case.